### DISCUSSION TITLE
How can I use raw-loader for .md files with webpack 5?

### DISCUSSION THREAD

**Discussion by beumsk** [webpack/webpack#16017] [Q&A]
Hello, 

I want to update my nextjs project from webpack 4 to v5. I need webpack to get my .md files and I was doing that in v4 with 'raw-loader'. However it is deprecated in v5 and I can't get this working.

See how my next.config.js was with webpack 4:
```
module.exports = {
  reactStrictMode: true,
  webpack5: false,
  target: 'serverless',
  eslint: {
    ignoreDuringBuilds: true,
  },
  webpack: function (config) {
    config.module.rules.push({ test: /\.md$/, use: 'raw-loader' });
    config.module.rules.push({ test: /\.yml$/, use: 'raw-loader' });
    return config;
  },
};
```

And how it is supposed to work with webpack 5 (according to this link: https://webpack.js.org/guides/asset-modules/):
```
module.exports = {
  reactStrictMode: true,
  target: 'serverless',
  eslint: {
    ignoreDuringBuilds: true,
  },
  module: {
    rules: [
      {
        test: /\.md$/,
        type: 'asset/source',
      },
    ],
  },
};
```

Here is the error message I am receiving for every .md file:
> hot-dev-client.js:120 ./posts/fix-the-window-is-not-defined-error.md
Module parse failed: Assigning to rvalue (1:2)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders

Can anyone guide me in the right direction? I spent countless hours searching the web and testing solutions...

Don't hesitate to ask more info if needed. Here is the repo link in case: https://github.com/beumsk/beumsk.github.io/blob/main/next.config.js.

Thank you :)

**Comment by alexander-akait** [comment]
Sounds like your loader configuration was not applied...
